src/shim/README.md

```diff
@@ -0,0 +1,34 @@
+# containerd-shim-rune-v2
+
+containerd-shim-rune-v2 is a shim for Enclave CC.
+
+## Introduction
+
+![shim-rune](docs/images/shim-rune.png)
+
+## Build requirements
+
+Go 1.13.x or above.
+
+## How to build and install
+
+### Step 1: Build and install shim binary.
+
+```bash
+git clone https://github.com/confidential-containers/enclave-cc
+
+cd src/shim
+GOOS=linux make binaries
+make install
```

Comment:
- This is an emotive topic, but I do wonder if we should use the Kata approach of always specifying a `$` prefix for commands the user should type:

  - https://github.com/kata-containers/kata-containers/blob/main/docs/Documentation-Requirements.md#code-blocks

  That makes it clear which lines are commands and which are command output, but does mean of course that you can't simply copy/paste the block into a terminal (although there are lots of standard JS widgets to toggle the prefix, but sadly GH doesn't offer that feature still).
- The `make install` will fail as written so I'd consider making that `sudo make install`.